src/css/table.css

```diff
@@ -17,13 +17,16 @@ tr.expanded {
   background-color: #131313;
 }
 
+tr.summary-row {
```

Comment:
Let's not forget to change this to `.table-container > table tr.sumary-row` when we merge our branches